afl_utils/afl_sync.py

```diff
@@ -163,6 +169,22 @@ def main(argv):
         print_err('Sorry, unknown command requested!')
         sys.exit(1)
 
+    rsync_put_options = _rsync_default_options[:]
+    rsync_get_options = _rsync_default_options[:]
+
+    if args.chmod or args.chown:
+        # these arguments are meaningless with pull since they should only
+        # affect the remote side
+        if args.cmd == 'pull':
+            print_err('Sorry, --chmod and --chown are meaningless with pull.')
```

Comment:
Maybe it makes sense to not bail out with a fatal error here. Printing a warning and ignoring the parameters may suffice. This would allow for 'command line history reuse' where `push` is substituted with `pull`.